addons/website_sale_delivery/views/website_sale_delivery.xml

```diff
@@ -50,5 +50,52 @@
+            <select name="shipping_country_id" class="form-control" t-att-disabled="'disabled' if shipping_id &gt;= 0 else ''">
+                <option value="">Country...</option>
+                <t t-foreach="shipping_countries or []" t-as="country">
+                    <option t-att-value="country.id" t-att-selected="country.id == checkout.get('shipping_country_id')"><t t-esc="country.name"/></option>
+                </t>
+            </select>
+        </xpath>
+        <xpath expr="//select[@name='country_id']" position="replace">
+            <select name="country_id" class="form-control js_country_all" t-att-style="not shipping_id and 'display:none' or ''" t-att-disabled="not shipping_id and True or False">
+                <option value="">Country...</option>
+                <t t-foreach="countries or []" t-as="country">
+                    <option t-att-value="country.id" t-att-selected="country.id == checkout.get('country_id')"><t t-esc="country.name"/></option>
+                </t>
+            </select>
+            <select name="country_id" class="form-control js_country_shipping" t-att-style="shipping_id and 'display:none' or ''" t-att-disabled="shipping_id and True or False">
+                <option value="">Country...</option>
+                <t t-foreach="shipping_countries or []" t-as="country">
+                    <option t-att-value="country.id" t-att-selected="country.id == checkout.get('country_id')"><t t-esc="country.name"/></option>
+                </t>
+            </select>
+        </xpath>
+        <xpath expr="//select[@name='shipping_state_id']" position="replace">
+            <select name="shipping_state_id" class="form-control" t-att-disabled="'disabled' if shipping_id &gt;= 0 else ''">
+                <option value="">State / Province...</option>
+                <t t-foreach="shipping_states or []" t-as="state">
+                    <option t-att-value="state.id" style="display:none;" t-att-data-country_id="state.country_id.id" t-att-selected="state.id == checkout.get('shipping_state_id')"><t t-esc="state.name"/></option>
+                </t>
+            </select>
+        </xpath>
+        <xpath expr="//select[@name='state_id']" position="replace">
+            <select name="state_id" class="form-control js_country_all" t-att-style="not shipping_id and 'display:none' or ''" t-att-disabled="not shipping_id and True or False">
+                <option value="">Select...</option>
+                <t t-foreach="states or []" t-as="state">
+                    <option t-att-value="state.id" style="display:none;" t-att-data-country_id="state.country_id.id" t-att-selected="state.id == checkout.get('state_id')"><t t-esc="state.name"/></option>
+                </t>
+            </select>
+            <select name="state_id" class="form-control js_country_shipping" t-att-style="shipping_id and 'display:none' or ''" t-att-disabled="shipping_id and True or False">
```

Comment:
Use either t-att-disabled="'disabled' ..." or t-att-disable="True ...", not both.